### ISSUE TITLE
Live_grep crashes neovim

### ISSUE THREAD

**Issue by jjalonsoc** [nvim-telescope/telescope.nvim#1530]
### Description

I used the minimal configuration provided below as my init.lua. Making any other file obsolete.  I try to run
`Telescope live_grep<CR>`
Type a random word in Live Grep and press enter. Instead of opening the file which shows correctly the result of the searched word, Neovim collapes.
![telescope_error](https://user-images.githubusercontent.com/32925200/144321498-93a28c8f-ade3-49b9-924e-69eecee721f8.png)
 

### Neovim version

```markdown
NVIM v0.6.0-dev+22-gaf263711a
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

Ubuntu 20.04.3 LTS

### checkhealth telescope

```markdown
health#telescope#check
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - WARNING: nvim-treesitter not found. 
 
## Checking external dependencies
  - OK: rg: found ripgrep 11.0.2
  - OK: fd: found fd 7.4.0
 
## ===== Installed extensions =====
 
## Telescope Extension: `fzf`
  - INFO: No healthcheck provided
```

### Steps to reproduce

1. nvim init.lua
2. :Telescope live_grep <Enter>
3. "text_example" <Enter>

### Expected behavior

To open the file that was appearing in the search

### Actual behavior

Neovim collapeses

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('fzf')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by fdschmidt93** [comment]
Unclear to me from what's described here as to why that might be happening off of the top of my head. Some ideas:

* Is everything (`nvim`, `telescope.nvim`, `plenary.nvim`, `telescope-fzf-native`) on the latest versions? `nvim` version seems dated, long shot but maybe there is something that's causing the issue.
* Does it only happen with `grep` pickers? `ripgrep` 13 introduced some changes https://github.com/nvim-telescope/telescope.nvim/issues/907, again a bit of a long whirl, but updating to `ripgrep` 13 might help
* Does it happen with an older commit (say 1-2 months old)? If not, can you `git bisect` the issue?

**Comment by tjdevries** [comment]
You must either the released version of nvim 0.6 or use nvim nightly.

Additionally, try removing fzf-native and then installing it again. Perhaps something changed w/ a LuaJIT version or something else that is causing problems.

**Comment by jjalonsoc** [comment]
Thanks, @tjdevries, and @fdschmidt93 for your answers, super helpful and fast! Another reason why I love this project. The problem was the neovim version I was using:
`NVIM v0.6.0-dev+22-gaf263711a`
After installing neovim from its stable branch, to the actual stable 0.6.0 Everything is working correctly. Thanks a lot
